spec/controllers/users_controller_spec.rb

```diff
@@ -24,7 +24,7 @@
         }
       }
       expect(response.status).to eq 201
-      expect(response.body).to be_json_eql({
+      expect(response.body).to include_json({
```

Comment:
Lets keep using `be_json_eql`, but also add the token key to the object.

We don't know what it is, but we can look it up with `User.first.token`